docs/contributing/frontend/testing.mdx

```diff
@@ -13,6 +13,20 @@ empty pipelines, triggers, etc. Set up your
 - Make sure [user authentication](https://docs.mage.ai/production/authentication/overview) is enabled.
 - Make sure your development server is running.
 
+## Configurating
+
+- Tests are default to fail if a client error occurs. Adding the following to the top of the test file to disable the check.
```

Comment:
Reword: `Tests default to "fail" if a client error occurs. Add the following to the top of the test file to disable this check.`